### DISCUSSION TITLE
Set maximum size allowed for an app to have

### DISCUSSION THREAD

**Discussion by szfkamil** [asmvik/yabai#2167] [General]
I have an Ultrawide monitor and for some apps windows stretched alone on a space looks bad.

There are window properties that can be manipulated here. This was some logic I could think of:

If it's detected that the window is alone on a space, it should be re-sized to a specific size and not allowed to go past that size… But, wouldn't yabai re-size the window unless it floats? If it floats, then if a new window is detected on the space, it should get back to being managed? 

Is yabai able to accomplish what I have in mind? Any suggestions?

**Comment by asmvik** [comment]
I will start thinking about ultrawide monitors in the near future. My current thoughts for how that would work align closely with #1888
